src/pykeen/models/inductive/inductive_nodepiece_gnn.py

```diff
@@ -106,9 +106,12 @@ def _get_representations(
         h: Optional[torch.LongTensor],
         r: Optional[torch.LongTensor],
         t: Optional[torch.LongTensor],
+        invert_relation: bool = False,  # TODO: do we need this here?
```

Comment:
might be useful to make sure there's no misatch in expectation when calling various functions